src/actions/collectiblesActions.js

```diff
@@ -175,17 +173,6 @@ const parseCollectibleTransactionFromOpenSeaHistoryItem = (event: OpenSeaHistory
   };
 };
 
-const isOpenSeaCollectibleTransaction = (event: Object): boolean => {
```

Comment:
Can we check with Etherspot that same logic has been adopted on their end?

Reply:
okay i think it is but will make sure